src/utils/ComponentUtils.ts

```diff
@@ -1,5 +1,7 @@
 import {ILinkSvgProps} from '../components/svg/LinkSvg';
 
+export type ComponentId = string;

```

Comment:
Is this new type really necessary? I'm not sure I see the added benefit since we are used to see ids being strings anyway.

Reply:
i personally think it reads better in the code and it's easier to follow the logic but can revert if majority thinks like you